src/Entity/Incident/Incident.php

```diff
@@ -0,0 +1,237 @@
+<?php
+
+namespace AppBundle\Entity\Incident;
+
+use ApiPlatform\Core\Annotation\ApiResource;
+use AppBundle\Entity\Model\TaggableInterface;
+use AppBundle\Entity\Model\TaggableTrait;
+use AppBundle\Entity\Task;
+use AppBundle\Entity\User;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use AppBundle\Action\Incident\CreateComment;
+use AppBundle\Action\Incident\IncidentAction;
+use AppBundle\Action\Incident\IncidentFastList;
+use AppBundle\Action\Incident\CreateIncident;
+use Symfony\Component\Serializer\Annotation\Groups;
+
+
+/**
+ * @ApiResource(
+ *   collectionOperations={
+ *     "get"={
+ *       "method"="GET",
+ *       "controller"=IncidentFastList::class,
+ *     },
+ *     "post"={
+ *       "method"="POST",
+ *       "controller"=CreateIncident::class,
+ *     }
+ *   },
+ *   itemOperations={
+ *     "get"={
+ *       "method"="GET",
+ *     },
+ *     "patch"={
+ *       "method"="PATCH",
+ *     },
+ *     "put"={
+ *       "method"="PUT",
+ *     },
+ *     "add_comment"={
+ *       "method"="POST",
+ *       "path"="/incidents/{id}/comments",
+ *       "controller"=CreateComment::class,
+ *     },
+ *     "action"={
+ *       "method"="PUT",
+ *       "path"="/incidents/{id}/action",
+ *       "controller"=IncidentAction::class,
+ *     }
+ *   }
+ * )
+ */
+class Incident implements TaggableInterface {
+    use TaggableTrait;
+
+    /**
+    * @Groups({"incident"})
+    */
```

Comment:
Wrong PHP docblock indentation. It can prevent PHP reflection from working as expected.